Transformers/Transformers.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>Are we training the language models here? I don't know because there are no comments in the notebook. </p><p> </p><p>Also, where do the params come from? No clue. </p><p> </p><p>I am personally not a huge fan of running too many scripts directly from the terminal. Is there any decent way to just call the driver function by importing it instead? </p><p> </p><p>And ohh, separate cells please? So that we don't re-run the previous command every time we need a model which is later in the cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Run language modelling is from the script <a href="huggingface/transformers/examples/language-modeling" target="_blank">here</a>. </p><p>Added comments to explain what each command line arg is doing in <a href="https://github.com/NirantK/Hinglish/commit/1d06aa2c1c80324d69866b1db335a5981bb84a9f" target="_blank">1d06aa2</a></p><p>and split the cells in <a href="https://github.com/NirantK/Hinglish/commit/f854e4b2700192d09a277b40480c1510597c20f9" target="_blank">f854e4b</a></p>